src/Neo/SmartContract/Contract.cs

```diff
@@ -25,14 +25,14 @@ public class Contract
         /// <summary>
         /// The script of the contract.
         /// </summary>
-        public byte[] Script;
+        public byte[] Script { get => field ??= []; set; }
```

Comment:
Here the logic it's different, we set in the script 

```suggestion
        public byte[] Script { get => field ?? []; set; }
```

Reply:
Agree but there are places where it's done and other where is not done, but we can unify this behaviour later